src/pool/rpc.h

```diff
@@ -366,6 +370,16 @@ CRT_RPC_DECLARE(pool_get_acl, DAOS_ISEQ_POOL_GET_ACL, DAOS_OSEQ_POOL_GET_ACL)
 
 CRT_RPC_DECLARE(pool_prop_set, DAOS_ISEQ_POOL_PROP_SET, DAOS_OSEQ_POOL_PROP_SET)
 
+#define DAOS_ISEQ_POOL_ACL_UPDATE	/* input fields */	 \
+	((struct pool_op_in)	(pui_op)		CRT_VAR) \
+	((struct daos_acl)	(pui_acl)		CRT_PTR)
```

Comment:
Does it make sense (add any value through symmetry?)  to change previously implemented DAOS_OSEQ_POOL_GET_ACL to also use struct daos_acl rather than daos_prop_t?

Reply:
I was planning to change DAOS_OSEQ_POOL_GET_ACL to a more generic DAOS_OSEQ_POOL_GET_PROP. I have a ticket to add the owner information to the output, which requires getting further pool properties. I'm not sure it's useful to have the underlying call be ACL specific - get_prop may be more useful to others as well.